autoload/multiple_cursors.vim

```diff
@@ -54,6 +54,13 @@ if !hlexists(s:hi_group_visual)
   exec "highlight link ".s:hi_group_visual." Visual"
 endif
 
+" List of operations (later compared with s:char[0]) that should trigger saving
+" of paste buffer for each cursor. The idea is to change cursor's paste buffer
+" only when operation is in fact changing unnamed cursor. Checking like this is
+" ugly and there should be a better, reliable and more portable way to do that.
+" This list is only used for checks in Normal mode.
+let s:paste_buffer_supported_operations = 'cCsSxXdD'
```

Comment:
Hm, no, it's not. I probably just forgot about that. I've just tried to add `yY` here and `y` to `normal_maps` and it seems to work. Good catch, thanks!